### ISSUE TITLE
Conda 4.13.0 is not supported

### ISSUE THREAD

**Issue by PekkaSavolainen** [spine-tools/Spine-Toolbox#1738]
**TLDR**: Use Conda 4.12.0 or older, don't use 4.13.0.

Trying to execute (on Win10) a Python Tool in a Conda environment with conda version 4.13.0 does not work and produces this output to the app.

![conda_4_13_fail](https://user-images.githubusercontent.com/43600314/176894863-79a084b8-6619-4f58-91c4-0a31aee4bc4e.png)

Here's `conda list` in **base**

```
(base) C:\>conda list
# packages in environment at C:\ProgramData\Miniconda3:
#
# Name                    Version                   Build  Channel
brotlipy                  0.7.0           py39h2bbff1b_1003
ca-certificates           2022.4.26            haa95532_0
certifi                   2022.6.15        py39haa95532_0
cffi                      1.15.0           py39h2bbff1b_1
charset-normalizer        2.0.4              pyhd3eb1b0_0
colorama                  0.4.4              pyhd3eb1b0_0
conda                     4.13.0           py39haa95532_0
conda-package-handling    1.8.1            py39h8cc25b3_0
console_shortcut          0.1.1                         4
cryptography              37.0.1           py39h21b164f_0
idna                      3.3                pyhd3eb1b0_0
menuinst                  1.4.16           py39h2bbff1b_0
openssl                   1.1.1p               h2bbff1b_0
pip                       21.2.4           py39haa95532_0
powershell_shortcut       0.0.1                         3
pycosat                   0.6.3            py39h2bbff1b_0
pycparser                 2.21               pyhd3eb1b0_0
pyopenssl                 22.0.0             pyhd3eb1b0_0
pysocks                   1.7.1            py39haa95532_0
python                    3.9.5                h6244533_3
pywin32                   228              py39he774522_0
requests                  2.28.0           py39haa95532_0
ruamel_yaml               0.15.100         py39h2bbff1b_0
setuptools                61.2.0           py39haa95532_0
sqlite                    3.38.5               h2bbff1b_0
tqdm                      4.64.0           py39haa95532_0
tzdata                    2022a                hda174b7_0
urllib3                   1.26.9           py39haa95532_0
vc                        14.2                 h21ff451_1
vs2015_runtime            14.27.29016          h5e58377_2
wheel                     0.37.1             pyhd3eb1b0_0
win_inet_pton             1.1.0            py39haa95532_0
wincertstore              0.2              py39haa95532_2
yaml                      0.2.5                he774522_0
```

Here's `conda list` in environment **py310**. Created by `conda create -n py310 python=3.10` and then `pip install ipykernel`.

```
(py310) C:\>conda list
# packages in environment at C:\Users\ttepsa\.conda\envs\py310:
#
# Name                    Version                   Build  Channel
asttokens                 2.0.5                    pypi_0    pypi
backcall                  0.2.0                    pypi_0    pypi
bzip2                     1.0.8                he774522_0
ca-certificates           2022.4.26            haa95532_0
certifi                   2022.6.15       py310haa95532_0
colorama                  0.4.5                    pypi_0    pypi
debugpy                   1.6.0                    pypi_0    pypi
decorator                 5.1.1                    pypi_0    pypi
entrypoints               0.4                      pypi_0    pypi
executing                 0.8.3                    pypi_0    pypi
ipykernel                 6.15.0                   pypi_0    pypi
ipython                   8.4.0                    pypi_0    pypi
jedi                      0.18.1                   pypi_0    pypi
jupyter-client            7.3.4                    pypi_0    pypi
jupyter-core              4.10.0                   pypi_0    pypi
libffi                    3.4.2                hd77b12b_4
matplotlib-inline         0.1.3                    pypi_0    pypi
nest-asyncio              1.5.5                    pypi_0    pypi
openssl                   1.1.1p               h2bbff1b_0
packaging                 21.3                     pypi_0    pypi
parso                     0.8.3                    pypi_0    pypi
pickleshare               0.7.5                    pypi_0    pypi
pip                       21.2.4          py310haa95532_0
prompt-toolkit            3.0.30                   pypi_0    pypi
psutil                    5.9.1                    pypi_0    pypi
pure-eval                 0.2.2                    pypi_0    pypi
pygments                  2.12.0                   pypi_0    pypi
pyparsing                 3.0.9                    pypi_0    pypi
python                    3.10.4               hbb2ffb3_0
python-dateutil           2.8.2                    pypi_0    pypi
pywin32                   304                      pypi_0    pypi
pyzmq                     23.2.0                   pypi_0    pypi
setuptools                61.2.0          py310haa95532_0
six                       1.16.0                   pypi_0    pypi
sqlite                    3.38.5               h2bbff1b_0
stack-data                0.3.0                    pypi_0    pypi
tk                        8.6.12               h2bbff1b_0
tornado                   6.1                      pypi_0    pypi
traitlets                 5.3.0                    pypi_0    pypi
tzdata                    2022a                hda174b7_0
vc                        14.2                 h21ff451_1
vs2015_runtime            14.27.29016          h5e58377_2
wcwidth                   0.2.5                    pypi_0    pypi
wheel                     0.37.1             pyhd3eb1b0_0
wincertstore              0.2             py310haa95532_2
xz                        5.2.5                h8cc25b3_1
zlib                      1.2.12               h8cc25b3_2
```

To circumvent the problem, downgrade conda to 4.12 or older.

```
(base) C:\>conda install conda==4.12
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: C:\ProgramData\Miniconda3

  added / updated specs:
    - conda==4.12

The following packages will be DOWNGRADED:

  conda                               4.13.0-py39haa95532_0 --> 4.12.0-py39haa95532_0

Proceed ([y]/n)? y

Preparing transaction: done
Verifying transaction: done
Executing transaction: done

(base) C:\>
```

Now, executing the same Python Tool looks like this

![conda_4_12_success](https://user-images.githubusercontent.com/43600314/176897864-95ee7a4e-fcbe-45fc-bd17-f39779b0cd7f.png)

Tried debugging this in every conceivable way but could not figure out the problem. I suppose we have to wait for a new version of Conda and try again.

**Comment by PekkaSavolainen** [comment]
Updated my (mini)conda to 23.5 and tried running a Python Tool using a Conda environment kernel. It works fine now. Also, the detached consoles work fine.

![issue_1738_conda_support](https://github.com/spine-tools/Spine-Toolbox/assets/43600314/8adfeeef-3369-4d03-b02a-5332d1c65853)
